### PR TITLE
Updated user agent cache size

### PR DESCRIPTION
**Pull request by sarthak77**
Updated user agent cache size

### CODE REVIEW

File: hypertrace-trace-enricher/hypertrace-trace-enricher-impl/src/main/java/org/hypertrace/traceenricher/enrichment/enrichers/UserAgentSpanEnricher.java [hunk 1]
```diff
@@ -30,7 +30,7 @@ public class UserAgentSpanEnricher extends AbstractTraceEnricher {
 
   private static final String CACHE_CONFIG_KEY = "cache";
   private static final String CACHE_CONFIG_MAX_SIZE = "maxSize";
-  private static final int CACHE_MAX_SIZE_DEFAULT = 10000;
+  private static final int CACHE_MAX_SIZE_DEFAULT = 20000;
```
Comment (pavan-traceable) [inline hypertrace-trace-enricher/hypertrace-trace-enricher-impl/src/main/java/org/hypertrace/traceenricher/enrichment/enrichers/UserAgentSpanEnricher.java:33]: Where is the consumption to take from config ?

Comment (sarthak77) [reply hypertrace-trace-enricher/hypertrace-trace-enricher-impl/src/main/java/org/hypertrace/traceenricher/enrichment/enrichers/UserAgentSpanEnricher.java:33]: This was already present in the code. Just added in conf files.

File: hypertrace-trace-enricher/helm/values.yaml [hunk 2]
```diff
@@ -97,6 +97,8 @@ traceEnricherConfig:
   attributeServicePort: 9012
   configServiceHost: config-service
   configServicePort: 50101
+  userAgentSpanEnricherConfig:
+    userAgentCacheSize: 20000
```
[the remark this answers is not in the document]
Comment (sarthak77) [reply hypertrace-trace-enricher/helm/values.yaml:101]: It is not present currently. What should be the default values for expiry and refresh?

[the remark this answers is not in the document]
Comment (pavan-traceable) [reply hypertrace-trace-enricher/helm/values.yaml:101]: You can make refresh as 5 mins and expiry as 10 mins

[the remark this answers is not in the document]
Comment (sarthak77) [reply hypertrace-trace-enricher/helm/values.yaml]: added

**Review by pavan-traceable — APPROVED** [review]

**Review by skjindal93 — APPROVED** [review]

**Pull request merged**